src/components/common/ActionModal.vue

```diff
@@ -17,6 +13,30 @@
           <i class="material-icons">close</i>
         </div>
       </div>
+      <!-- This step stuff should be refactored into something smaller -->
+      <div class="step--container">
+        <StepComponent
+          step-title="Details"
+          :step="step"
+          step-name="txDetails"
+          step-number="1"
+          :include-line="true"
+        ></StepComponent>
+        <StepComponent
+          step-title="Fees"
```

Comment:
maybe 2nd step can be called `Review` instead of fees? We'll add the `TmAnyTx` component to the second step - and then the user will be reviewing their transaction and hitting next if they want to "Submit". 

What do you think @colw @faboweb?

Reply:
How would this be called in classic e-commerce? "Overview", "Checkout"?

Reply:
I think the whole flow would be called a checkout flow. I think `Overview` could be good. Review is also okwme because it is the action the user is supposed to take.

I feel pretty good about `Details` `Review` `Submit` but certainly open to better options!

Reply:
Summary also works.

Reply:
`Details / Review / Submit`, but perhaps the `Details` could be set by the caller modal.

Reply:
Hahahaha I think we should lean into the "shopping" domain. People understand this.

Reply:
How about doing 4 steps? <Action>/Review/Sign/Submit ?

Reply:
how / why would sign and submit be separate?

Reply:
sign is the step where the user signs the transactions / picks a method and waits for the signature to be available
submit is the step where the user waits for the tx so be submitted and finally included in a block

Reply:
ok let's explore this more but not in this PR